nailgun/entities.py

```diff
@@ -6792,7 +6792,7 @@ def __init__(self, server_config=None, **kwargs):
             'jwt_expiration': entity_fields.IntegerField(default=4),
             'repo': entity_fields.StringField(default=''),
             'repo_gpg_key_url': entity_fields.URLField(default=''),
-            'packages': entity_fields.ListField(default=[]),
+            'packages': entity_fields.StringField(),
```

Comment:
@dosas Good catch, this field should only be a string field, but robottelo uses it as a list field only, will you be opening a PR for that too?

Reply:
Shouldn't some tests be failing if packages was passed?

I cannot find any usage of packages in robottelo tests. 

`RegistrationCommand` is called by `api_register` : https://github.com/search?q=repo%3ASatelliteQE%2Frobottelo%20api_register&type=code

Reply:
ping @Gauravtalreja1

Reply:
ah, right, we've no registration tests for api's to check if packages works

Reply:
Created PR to cover this in robottelo test - https://github.com/SatelliteQE/robottelo/pull/16914